spec.bs

```diff
@@ -168,6 +168,42 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
 
   When the [=user agent/configuration version=] is updated, the browser must properly migrate or delete data in [=user agent/user topics state=] and [=user agent/topics history storage=] so that the state and the configuration are consistent.
 
+  <h3 id="expiring stored data">Expiring stored data</h3>
+  User agents may automatically delete stored data after a period of time.
```

Comment:
Let's change this one to must. I think it's important that browsers are consistent here. Then remove "Chrome" below and just supply the algorithm.

Reply:
Done.